playwrighte2e/steps/accessibilitySteps.ts

```diff
@@ -79,8 +79,8 @@ export default class AccessibilitySteps extends BaseStep {
       accessibilityEnabled,
       axeUtils,
     );
-    await this.caseListPage.searchCaseApplicationWithSubmissionReference('Eng/Wales - Singles', subRef);
-    await this.caseListPage.processCaseFromCaseList();
+
+    await  this.caseListPage.navigateToCaseDetails(subRef, 'EnglandWales')
```

Comment:
minor - double space
```suggestion
    await this.caseListPage.navigateToCaseDetails(subRef, 'EnglandWales')
```